http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200701_kubernetes_drop_port_table.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="rtitle" id="kubernetes_drop_port_table">
+        <dropTable cascadeConstraints="true" tableName="PORT"/>
```

Comment:
you may need to explicitly drop the FK constraints before dropping the table 🤔

Reply:
Maybe - I was wondering if `cascadeConstraints` will work. Will test

Reply:
This seems to successfully drop the table and leave to constraints behind